mypy/fastparse.py

```diff
@@ -513,6 +516,7 @@ def do_func_def(self, n: Union[ast3.FunctionDef, ast3.AsyncFunctionDef],
             func_type.definition = func_def
             func_type.line = lineno
 
+        retval: Union[FuncDef, Decorator]
```

Comment:
Because mypy needs to run with Python 3.5, you can't use this PEP-526 syntax here yet (which is 3.6 or later only). To fix this, use a type comment on the first line that assigns to `retval`.

Reply:
Sorry, fixing now.